Sources/CvUnit.cpp

```diff
@@ -5235,15 +5235,12 @@ bool CvUnit::isActionRecommended(int iAction)
 		pPlot = plot();
 	}
 
-// BUFFY - Don't Recommend Actions in Fog of War - start
-#ifdef _BUFFY
-	// from HOF Mod - Denniz
-	if (!pPlot->isVisible(GC.getGameINLINE().getActiveTeam(), false))
-	{
-		return false;
-	}
-#endif
-// BUFFY - Don't Recommend Actions in Fog of War - end
+	// Don't Recommend Actions in Fog of War - start
```

Comment:
Same question, is it better with this code or not?

Reply:
No idea. It's probably never been on. It's up to TB and the others if it's better on or not.